internal/pkg/service/appsproxy/proxy/proxy_test.go

```diff
@@ -2105,6 +2105,7 @@ func testDataApps(upstream *url.URL, m []*mockoidc.MockOIDC) []api.AppConfig {
 			ID:             "12345",
 			ProjectID:      "123",
 			Name:           "LOWERCASE",
+			AppSlug:        pointer("LOWERCASE"),
```

Comment:
Please, in tests, use some different app name as the `slug`, so it will be clear that the `Name` is no more used in URL.

All apps can have for example name `My App`.